webpack.config.js

```diff
@@ -125,13 +125,18 @@ module.exports = {
   resolve: {
     extensions: ['.webpack.js', '.web.js', '.ts', '.tsx', '.js', '.json', '.css', '.scss'],
     alias: {
-      Configuration: path.resolve(configurationPath)
+      Configuration: path.resolve(configurationPath),
+      jsonld$: path.resolve(__dirname, './node_modules/jsonld/dist/jsonld.min.js')
```

Comment:
@lautarodragan @krobi64 When using @po.et/poet-js package I'm running into several problems. First the json-ld package was causing the error: 

```
ERROR in ./node_modules/jsonld/lib/jsonld.js
Module parse failed: Unexpected token (1040:2)
You may need an appropriate loader to handle this file type.
| function _setDefaults(options, {
|   documentLoader = jsonld.documentLoader,
|   ...defaults
| }) {
|   if(typeof options === 'function') {
 @ ./src/utils/JSONLD.ts 3:15-32
 @ ./src/index.ts
Warning:  Use --force to continue.

Aborted due to warnings.
```

I was able to fix it with this line, which I got from the issues page: https://github.com/digitalbazaar/jsonld.js/issues/252#issuecomment-417816242